src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,127 @@
+package duke.ui;
+
+import duke.task.Task;
+
+import java.util.List;
+
+/**
+ * A class that contains methods to print responses to the screen.
+ */
+public class Ui {
+    private String lineBreak = "-------------------------------\n";
+    private String welcomeMessage = "Hello! I'm Duke\nWhat can I do for you?\n";
+    private String goodbyeMessage = "Bye. Hope to see you again soon!";
+
+    /**
+     * Prints welcome message to the screen.
+     */
+    public void greet() {
+        System.out.println(lineBreak + welcomeMessage + lineBreak);
+    }
+
+    /**
+     * Prints goodbye message to the screen.
+     */
+    public void goodbye() {
+        System.out.println(goodbyeMessage);
+    }
+
+    /**
+     * Prints line divider to the screen.
+     */
+    public void showLine() {
+        System.out.println(lineBreak);
+    }
+
+    /**
+     * Prints the successful task added message to the screen.
+     *
+     * @param task Task that was added.
+     * @param noOfTasks Number of tasks that are currently in the list of tasks.
+     */
+    public void taskAddedMessage(Task task, int noOfTasks) {
+        System.out.println("Got it. I've added this task:\n"
+                + task.toString()
+                + "\nNow you have " + noOfTasks + " tasks in the list.");
+    }
+
+    /**
+     * Prints out the contents of a list of tasks.
+     *
+     * @param tasks List of tasks to be printed.
+     */
+    public void list(List<Task> tasks) {
```

Comment:
Would naming this `printList` align with the code guidelines better?

Reply:
Youre right! 🦖